src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -352,6 +357,9 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
                     </>
                   </NavItem>
                 ))}
+              { this.state.serviceDetailsInfo.apiDocumentation && this.state.serviceDetailsInfo.apiDocumentation.baseUrl && (
```

Comment:
baseUrl should be not `''` and not `undefined`, empty strings are not false in javascript, as far as I know, and if they are, that's black magic :P

Reply:
Let's fight black magic then, I added !== ''